core/lib/libtomcrypt/src/tee_ltc_provider.c

```diff
@@ -2402,6 +2424,10 @@ struct crypto_ops crypto_ops = {
 		.dsa_sign = dsa_sign,
 		.dsa_verify = dsa_verify,
 #endif
+#if defined(CFG_CRYPTO_ECC)
+		/* temporary dummy function update next patch */
+		.alloc_ecc_keypair = alloc_ecc_keypair_dummy,
+#endif
```

Comment:
Why add this function only? If you want to implement ECC support in several patches, it's probably OK but I think the patches should be split differently.